libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -142,6 +143,10 @@ class AppPrompt {
     //   if (!e.target.closest(CONFIG.selectors.prompt)) AppPrompt.close();
     // });
 
+    document.addEventListener('keydown', (event) => {
+      if (['Tab', 'Escape', 'Enter'].includes(event.key)) this.close();
+    });
```

Comment:
We should IMO only attach/re-attach this when the modal is open.

Reply:
If the logic reaches this point, the modal should be open. However, we should remove the event listener when the modal is closed. If the user clicks on any close button, we don't have any need for this event listener, since there won't be any prompt to close

Reply:
True, better remove the event listener when PEP is closed. Made the change 👍